pkg/resources/resource_monitor.go

```diff
@@ -2,504 +2,450 @@ package resources
+		Description:      "The date and time when the resource monitor starts monitoring credit usage for the assigned warehouses. If you set a `start_timestamp` for a resource monitor, you must also set `frequency`.  After removing this field from the config, the previously set value will be preserved on the Snowflake side, not the default value. That's due to Snowflake limitation and the lack of unset functionality for this parameter.",
 	},
 	"end_timestamp": {
-		Type:        schema.TypeString,
-		Optional:    true,
-		Description: "The date and time when the resource monitor suspends the assigned warehouses.",
-	},
-	"suspend_trigger": {
-		Type:          schema.TypeInt,
-		Optional:      true,
-		Description:   "The number that represents the percentage threshold at which to suspend all warehouses.",
-		ConflictsWith: []string{"suspend_triggers"},
-	},
-	"suspend_triggers": {
-		Type:          schema.TypeSet,
-		Elem:          &schema.Schema{Type: schema.TypeInt},
-		Optional:      true,
-		Description:   "A list of percentage thresholds at which to suspend all warehouses.",
-		ConflictsWith: []string{"suspend_trigger"},
-		Deprecated:    "Use suspend_trigger instead",
-	},
-	"suspend_immediate_trigger": {
-		Type:          schema.TypeInt,
-		Optional:      true,
-		Description:   "The number that represents the percentage threshold at which to immediately suspend all warehouses.",
-		ConflictsWith: []string{"suspend_immediate_triggers"},
-	},
-	"suspend_immediate_triggers": {
-		Type:          schema.TypeSet,
-		Elem:          &schema.Schema{Type: schema.TypeInt},
-		Optional:      true,
-		Description:   "A list of percentage thresholds at which to suspend all warehouses.",
-		ConflictsWith: []string{"suspend_immediate_trigger"},
-		Deprecated:    "Use suspend_immediate_trigger instead",
+		Type:             schema.TypeString,
+		Optional:         true,
+		DiffSuppressFunc: IgnoreChangeToCurrentSnowflakeValueInShow("end_time"),
+		Description:      "The date and time when the resource monitor suspends the assigned warehouses.",
 	},
 	"notify_triggers": {
 		Type:        schema.TypeSet,
-		Elem:        &schema.Schema{Type: schema.TypeInt},
 		Optional:    true,
-		Description: "A list of percentage thresholds at which to send an alert to subscribed users.",
+		Description: "Specifies a list of percentages of the credit quota. After reaching any of the values the users passed in the notify_users field will be notified (to receive the notification they should have notifications enabled). Values over 100 are supported.",
+		Elem: &schema.Schema{
+			Type:             schema.TypeInt,
+			ValidateDiagFunc: validation.ToDiagFunc(validation.IntAtLeast(1)),
+		},
 	},
-	"set_for_account": {
-		Type:        schema.TypeBool,
-		Optional:    true,
-		Description: "Specifies whether the resource monitor should be applied globally to your Snowflake account (defaults to false).",
-		Default:     false,
+	"suspend_trigger": {
```

Comment:
should these two be also added IgnoreChangeToCurrentSnowflakeValueInShow ?

Reply:
True, I also added test with external changes, but I'm not sure if that showed IgnoreChangeToCurrentSnowflakeValueInShow worked for those fields (I don't think so).